geoips/interfaces/yaml_based/sectors.py

```diff
@@ -12,13 +12,91 @@
 
 """Sector interface module."""
 
-from geoips.interfaces.base import BaseYamlInterface
+from geoips.interfaces.base import BaseYamlPlugin, BaseYamlInterface
+from pyresample import load_area
+from geoips.image_utils.mpl_utils import create_figure_and_main_ax_and_mapobj
+from cartopy import feature as cfeature
+from pyresample import create_area_def
+
+
+# Commenting these out for PR #260
+# Will work on this again after the 2023 workshop
```

Comment:
I added #301 to work on this. I'd like to leave these here as examples for #301.